lib/pbench/server/api/resources/db_models.py

```diff
@@ -0,0 +1,83 @@
+import jwt
+import datetime
+from dateutil import parser
+from pbench.server.api import app, bcrypt, db
+
+
+class UserModel(db.Model):
+    """ User Model for storing user related details """
+    __tablename__ = "users"
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    username = db.Column(db.String(255), unique=True, nullable=False)
+    firstName = db.Column(db.String(255), unique=False, nullable=False)
+    lastName = db.Column(db.String(255), unique=False, nullable=False)
+    password = db.Column(db.String(255), nullable=False)
+    registered_on = db.Column(db.DateTime, nullable=False)
+    # ToDo: decide if we want the email field
```

Comment:
Okay I will add the email field I think that make sense. Whats avatar image??

Reply:
I guess I should also add a regex checking for validating the email pattern

Reply:
We want to support all this directly from backend; the UI might well choose to do a pre-check because it's more efficient than the round-trip REST, but we shouldn't let `curl` create an invalid user context.